static/src/css/vendor/bootstrap/_custom-variables.scss

```diff
@@ -20,7 +20,7 @@ $modal-content-border-radius: 0;
 /* Theme Colors */
 $theme-colors: (
   "primary": utils.$color__nasa-blue,
-  "light": utils.$color__carbon--5,
+  "light": utils.$color__spacesuit-white,
```

Comment:
This will carry that problem we had in the layer-picker where the buttons on some things like the collection details and add projects are the same color as background.
Suggest reverting and moving that work to a standalone ticket